mithril-client/src/utils/unpacker.rs

```diff
@@ -90,12 +91,15 @@ impl SnapshotUnpacker {
 
     /// Unpack the snapshot pointed at the given filepath into the given directory.
     pub async fn unpack_snapshot(&self, filepath: &Path, unpack_dir: &Path) -> StdResult<()> {
-        let snapshot_file_tar_gz =
+        let mut snapshot_file_tar_gz =
             File::open(filepath).map_err(|e| SnapshotUnpackerError::UnpackFailed {
                 filepath: filepath.to_owned(),
                 dirpath: unpack_dir.to_owned(),
                 error: e.into(),
             })?;
+        // Try to force the file read to start at 0. We need to determine if
+        // this fixes instabilities in CI.
```

Comment:
Can you update this comment to explain why this is necessary and remove the reference to the CI ?